server/database/queries/deleteProductQuery.js

```diff
@@ -0,0 +1,5 @@
+const connection = require('../connection');
+
+const deleteProductQuery = (productId) => connection.query('DELETE FROM cart WHERE product_id = $1', [productId]);
```

Comment:
You should add another statement here for the user id in order to make sure that we are deleting a product from this specific user cart